lib/questions.js

```diff
@@ -0,0 +1,14 @@
+const readline = require("readline");
+const interface = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout
+  })
```

Comment:
this will probably lead to some bugs when using other functions than set

Reply:
Thank you. I fixed it.